src/modules/users/components/TokenActivation/TokenActivationContent/TokensTab.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useMemo } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { BigNumber } from 'ethers/utils';
+
+import Icon from '~core/Icon';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import Numeral from '~core/Numeral';
+
+import { UserToken } from '~data/generated';
+import { formatTokenValue } from '~utils/numbers';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './TokenActivationContent.css';
+import ChangeTokenStateForm from './ChangeTokenStateForm';
+import TokenTooltip from './TokenTooltip';
+
+const MSG = defineMessages({
+  active: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.active',
+    defaultMessage: 'Active',
+  },
+  inactive: {
+    id: 'users.TokenActivation.TokenActivationContent.TokensTab.inactive',
+    defaultMessage: 'Inactive',
+  },
+  activeTokensTooltip: {
+    id:
+      // eslint-disable-next-line max-len
+      'users.TokenActivation.TokenActivationContent.TokensTab.activeTokensTooltip',
```

Comment:
You could use back quotes( \` ) here to avoid having to use the `eslint-disable-next-line max-len`